scripts/battlefields/Riverne_Site_A01/ouryu_cometh.lua

```diff
@@ -0,0 +1,41 @@
+-----------------------------------
+-- Area: Riverne Site #A01
+-- Name: Ouryu Cometh
+-- !pos 732.550 -32.5 -506.544 30
+-----------------------------------
+local riverneID = zones[xi.zone.RIVERNE_SITE_A01]
+-----------------------------------
+
+local content = Battlefield:new({
+    zoneId        = xi.zone.RIVERNE_SITE_A01,
+    battlefieldId = xi.battlefield.id.OURYU_COMETH,
+    maxPlayers    = 18,
+    levelCap      = 75,
+    timeLimit     = utils.minutes(30),
+    index         = 0,
+    area          = 1,
+    entryNpc      = 'Unstable_Displacement',
+    exitNpc       = 'Spatial_Displacement',
+    requiredItems =
+    {
+        xi.item.CLOUD_EVOKER,
+        wearMessage = riverneID.text.TIME_LIMIT_FOR_THIS_BATTLE_IS + 2,
+        wornMessage = riverneID.text.TIME_LIMIT_FOR_THIS_BATTLE_IS + 1,
+    },
+    experimental = true,
+})
+
+content.groups =
+{
+    {
+        mobs      = { 'Ouryu' },
+        superlink = true,
```

Comment:
So just one last thing as I look with a fresh mind: Groups superlink to their own, so instead for here and below, you're going to need to use `superlinkGroup = 1` instead (can have multiple groups defined, but if they need to link together, need to be the same)